src/recipes/tests/mocks/recipe-query-params.mock.ts

```diff
@@ -0,0 +1,13 @@
+import { GetRecipesQueryParamsDto } from 'src/recipes/dto/get-recipe-query-params.dto';
+
+export const mockedGetRecipesQueryParams: GetRecipesQueryParamsDto = {
+  details: 'test',
+  categoryIds: ['1'],
+  rating: 3,
+  startDate: new Date(),
+  endDate: new Date(),
+  title: 'test',
+  description: 'test',
+  ingredient: 'test',
+  step: 'test',
```

Comment:
if you want you can use https://fakerjs.dev/ library which can generate fake data for you every time you run the tests